openpype/hosts/houdini/plugins/publish/collect_fbx_type.py

```diff
@@ -0,0 +1,59 @@
+"""Collector for filmboxfbx types.
+
+Collectors act as a pre process for the validation stage.
+It is used mainly to update instance.data
+
+P.S.
+    There are some collectors that run by default
+    for all types.
+
+This plugin is part of publish process guide.
+"""
+import pyblish.api
+
+
+class CollectFilmboxfbxType(pyblish.api.InstancePlugin):
+    """Collect data type for fbx instance."""
+
+    hosts = ["houdini"]
+    families = ["fbx", "staticMesh"]
+    label = "Collect type of fbx"
+
+    # Usually you will use this value as default
+    order = pyblish.api.CollectorOrder
+
+    # overrides InstancePlugin.process()
+    def process(self, instance):
+
+        if instance.data["creator_identifier"] == "io.openpype.creators.houdini.unrealstaticmesh":  # noqa: E501
+            # such a condition can be used to differentiate between
+            #  instances by identifier becuase sometimes instances
+            #  may have the same family but different identifier
+            #  e.g. bgeo and alembic
+            instance.data["families"] += ["fbx"]
+
+        # Update instance.data with ouptut_node
+        out_node = self.get_output_node(instance)
+
+        if out_node:
+            instance.data["output_node"] = out_node
+
+        # Disclaimer : As a convntin we use collect_output_node.py
+        #   to Update instance.data with ouptut_node of different types
+        #   however, this collector is used for demonstration
```

Comment:
So you're saying it's not needed, but why would you still like to "demonstrate" this here? I'm a bit unsure what you're asking here.

Reply:
I thought it would be simpler to understand if  written in  a separate plugin. 
but this doesn't help as you mentioned!